src/azure-cli/azure/cli/command_modules/containerapp/_params.py

```diff
@@ -161,7 +161,7 @@ def load_arguments(self, _):
 
     with self.argument_context('containerapp env', arg_group='Virtual Network') as c:
         c.argument('infrastructure_subnet_resource_id', options_list=['--infrastructure-subnet-resource-id', '-s'], help='Resource ID of a subnet for infrastructure components and user app containers.')
-        c.argument('docker_bridge_cidr', options_list=['--docker-bridge-cidr'], help='CIDR notation IP range assigned to the Docker bridge. It must not overlap with any Subnet IP ranges or the IP range defined in Platform Reserved CIDR, if defined')
+        c.argument('docker_bridge_cidr', options_list=['--docker-bridge-cidr'], help='CIDR notation IP range assigned to the Docker bridge. It must not overlap with any Subnet IP ranges or the IP range defined in Platform Reserved CIDR, if defined', deprecate_info=c.deprecate(target='--docker-bridge-cidr', hide=True))
```

Comment:
add target version?

Reply:
The latest version include this change is `2.65.0`, I will use `2.68.0` as the target expiration version:
![image](https://github.com/user-attachments/assets/aa84f895-fea6-467a-8dd5-a76ef6d84a5c)